### ISSUE TITLE
ToolProgression doesn't work with Pyrotech

### ISSUE THREAD

**Issue by EmeraldsEmerald** [codetaylor/pyrotech#164]
#### Issue Description

Any commands from the [ToolProgression](https://www.curseforge.com/minecraft/mc-mods/tool-progression) mod have no effect on Pyrotech tools. This applies to the config files as well.

#### What Happens

I tried to change the mining level of the Flint Pickaxe by holding it in my hand and typing the command `/tpr set pickaxe 0`

This had no effect, and kept the pickaxe's mining level at iron tier. After further testing, it turned out that no tools from the mod were affected by any command I tried at all. Changing the configs and even restarting the game had no effect either.

#### What You Expect to Happen

The mining level of the tool is changed to match the mining level specified in the command/config file.

#### Affected Versions

Do *not* use `latest`; please supply accurate version numbers.

* Minecraft: 1.12.2
* Forge: 14.23.5.2838
* Dropt: 1.12.2-1.15.1
* Pyrotech: 1.12.2-1.3.6
* Athenaeum: 1.12.2-1.17.3
* ToolProgression: 1.12.2-1.6.6

**Comment by codetaylor** [comment]
I'm not sure if this can be fixed, however, Pyrotech does have a config that let's you adjust the harvest level and durability of each tool material in `module.Tool.cfg`.

```
    ##########################################################################################################
    # harvest_level
    #--------------------------------------------------------------------------------------------------------#
    # The harvest levels of each tool material.
    ##########################################################################################################

    harvest_level {
        I:bone=1
        I:crude=0
        I:flint=1
        I:obsidian=2
    }
```

**Comment by codetaylor** [comment]
Fixed in 1.4.0
